.github/workflows/ci.yml

```diff
@@ -1039,3 +1039,82 @@ jobs:
           check_name: ${{ github.job }} with secrets
           conclusion: ${{ job.status }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
+
+  # TODO: Don't always run the Redshift tests; add impact analysis or integrate with above if possible.
+  redshift:
+    name: redshift (${{ matrix.modules }})
+    runs-on: ubuntu-latest
+    if: ${{ env.TRINO_AWS_ACCESS_KEY_ID != '' && env.TRINO_AWS_SECRET_ACCESS_KEY != '' }}
+    env:
+      REDSHIFT_AWS_REGION: us-east-2
+      REDSHIFT_AWS_AVAILABILITY_ZONE: us-east-2a
+      REDSHIFT_NODE_TYPE: dc2.large
+      REDSHIFT_DATABASE_NAME: testdb
+      REDSHIFT_USERNAME: admin
+      # see plugin/trino-redshift/README.md for details on how to setup the Amazon Redshift CI infrastructure
+      REDSHIFT_SUBNET_GROUP_NAME: cluster-subnet-group-trino-ci
+      REDSHIFT_IAM_ROLES: arn:aws:iam::894365193301:role/redshift-ci
+      REDSHIFT_VPC_SECURITY_GROUP_IDS: "[\"sg-055bbbc00509a0345\"]"
+      REDSHIFT_S3_TPCH_TABLES_ROOT: "s3://trino-ci-redshift-test"
```

Comment:
Those settings are correlated with TRINO_AWS_ACCESS_KEY_ID, so they should be defined in the repo settings, not the ci.yml file. For example, if i want to setup up Redshift tests on my fork of Trino, I should be able to do this without modifying ci.yml file.

In Trino repo settings these would be vars (not secrets), since they probably do not need to be secret.
(cc @nineinchnick @ppalucha on whether account id, vpc/sg ids should remain secret)